src/useLocalStore.ts

```diff
@@ -0,0 +1,18 @@
+import { action, observable } from "mobx"
+import { useState } from "react"
+import { isPlainObject } from "./utils"
+
+export function useLocalStore<T>(initializer: () => T): T {
+    return useState(() => {
+        const store: any = observable(initializer())
+        if (isPlainObject(store)) {
+            Object.keys(store).forEach(key => {
+                const value = store[key]
+                if (typeof value === "function") {
+                    store[key] = action(value.bind(store))
```

Comment:
Probably we should only bind, but not `action`, as it will kill derivations that use local functions to compute things

Reply:
I am not entirely sure I understand the problem here, but hey I never properly understood the exact action mechanics. Let's merge it like this and if it turns out to be a problem, we can remove it.